tests/e2e/containers/containers.go

```diff
@@ -251,47 +244,13 @@ func (m *Manager) ExecCmd(t *testing.T, containerName string, command []string,
 				return true
 			}
 
-			errBufString := errBuf.String()
-			// When a validator attempts to send multiple transactions in the same block, the expected sequence number
-			// will be thrown off, causing the transaction to fail. It will eventually clear, but what the following code
-			// does is it takes the expected sequence number from the error message, adds a sequence number flag with that
-			// number, and retries the transaction. This allows for multiple txs from the same validator to be committed in the same block.
-			if (errBufString != "" || outBuf.String() != "") && containerName != hermesContainerName {
-				// Check if the error message matches the expected pattern
-				errBufMatches := sequenceMismatchRegex.FindAllStringSubmatch(errBufString, -1)
-				outBufMatches := sequenceMismatchRegex.FindAllStringSubmatch(outBuf.String(), -1)
-				if len(errBufMatches) > 0 {
-					lastArg := command[len(command)-1]
-					if strings.Contains(lastArg, "--sequence") {
-						// Remove the last argument from the command
-						command = command[:len(command)-1]
-					}
-					expectedSequenceStr := errBufMatches[len(errBufMatches)-1][1]
-					expectedSequence, _ = strconv.Atoi(expectedSequenceStr)
-					modifiedCommand := append(command, fmt.Sprintf("--sequence=%d", expectedSequence))
-					// Update the command for the next iteration
-					command = modifiedCommand
-				} else if len(outBufMatches) > 0 {
-					lastArg := command[len(command)-1]
-					if strings.Contains(lastArg, "--sequence") {
-						// Remove the last argument from the command
-						command = command[:len(command)-1]
-					}
-					expectedSequenceStr := outBufMatches[len(outBufMatches)-1][1]
-					expectedSequence, _ = strconv.Atoi(expectedSequenceStr)
-					modifiedCommand := append(command, fmt.Sprintf("--sequence=%d", expectedSequence))
-					// Update the command for the next iteration
-					command = modifiedCommand
-				}
-			}
```

Comment:
All this logic is no longer needed! We now don't check sequence numbers in E2E, and instead just inc a memo field to ensure txs are different